cmake_msbuild.cmd

```diff
@@ -13,12 +17,6 @@ if NOT '%ERRORLEVEL%' == '0' exit /b 1
 set Platform=
 set __ProjectDir=
 
-:: Restore the Tools directory
-call %~dp0init-tools.cmd
```

Comment:
I'm just asking because I'm not familiar enough with all that buildtools does for our repo, but do we never have a case where this script could be called without having initialized tools? This uses the semaphore so it should be a no-op if left in here.

Reply:
init-tools should not be used for cmake use cases.  Since I am explicitly trying to eliminate build tools references and trying to make this support the cmake usage only, I deliberately removed it.

Reply:
Ok, if it's only used for generating the vcxproj then this is a good idea. I just wasn't sure if we had any native-build targets that come from this that any native build flavor would depend on :) But most likely that would be caught by a screaming CI.